tslint.json

```diff
@@ -0,0 +1,162 @@
+{
+    "extends": "tslint:recommended",
+    "rules": {
+        "indent": [
+          true,
+					"tabs"
```

Comment:
Lol when @petschekr comments with a :cry: it means he's sad, but it doens't mean you have to make him feel better.
I think the jurisdiction on style rests with @kexin-zhang now, but I don't _think_ she cares either way. I'd ask.